addons/fire/stringtable.xml

```diff
@@ -8,6 +8,7 @@
             <Russian>ACE Возгорание</Russian>
             <German>ACE Feuer</German>
             <Polish>ACE Ogień</Polish>
+            <Spanish>Fuego ACE</Spanish>
```

Comment:
I think this is for the setting's category I think it's best for all ace settings to have the same ace prefix so they are listed together in alphabetically.

```suggestion
            <Spanish>ACE Fuego</Spanish>
```